tasks/libs/pipeline/notifications.py

```diff
@@ -157,11 +158,12 @@ def email_to_slackid(ctx: Context, email: str) -> str:
 def warn_new_commits(release_managers, team, branch, next_rc):
     from slack_sdk import WebClient
 
+    today = datetime.today()
+    rc_date = datetime(today.year, today.month, today.day, hour=14, minute=0, second=0, tzinfo=timezone.utc)
+    rc_schedule_link = "https://github.com/DataDog/datadog-agent/blob/main/.github/workflows/create_rc_pr.yml#L6"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Maybe using a permalink instead as there is a line reference (this won't probably change soon but in case 🤷)